projects/packages/sync/src/modules/class-callables.php

```diff
@@ -389,7 +389,7 @@ public function set_plugin_action_links() {
 				$action_links = array();
 			}
 			$formatted_action_links = null;
-			if ( ! empty( $action_links ) && count( $action_links ) > 0 ) {
+			if ( ! empty( $action_links ) && array() !== $action_links ) {
```

Comment:
`! empty( $action_links )` is already false for the empty array, so both the `count()` check before and the `array() !==` check now are redundant.

For that matter the `! empty()` itself is a bit redundant too, because `isset( $action_links )` will always be true. So this could reduce to just `if ( $action_links ) {`.